### PR TITLE
DDP-6865 - set other comments length validation to maxLength=10000

### PR DESCRIPTION
**Pull request by evgeniipr**
## Context

Preliminary implementation of the validation for OTHER_COMMENTS: set maxLimit=1000.
After verification of this change it will be implemented a Study Builder custom task (patch) which sets the change directly to DB.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [x] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [x] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [x] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [x] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: study-builder/studies/pancan/i18n/en.conf [hunk 1]
```diff
@@ -713,6 +713,7 @@
 "hint_req_signature": "Full Name is required",
 "hint_req_yes_no": "Please choose yes or no"
 "hint_dob": "Please enter date of birth in MM DD YYYY format",
+"hint_comments_too_long": "Comments too long"
```
Comment (evgeniipr) [inline study-builder/studies/pancan/i18n/en.conf:716]: @MocanaAtBroad  please suggest the correct text for a OTHER_COMMENTS validation hint (both en/es):
`"hint_comments_too_long": "Comments too long"`

File: study-builder/studies/pancan/about-you.conf [hunk 2]
```diff
@@ -281,7 +281,26 @@
                   ]
                 }
               ]
-            }
+            },
+            "validations": [
+              {
+                "ruleType": "LENGTH",
+                "maxLength": 10000,
```
Comment (MocanaAtBroad) [inline study-builder/studies/pancan/about-you.conf:288]: Let's make this much shorter: 500 should be more than enough.

File: study-builder/studies/pancan/i18n/en.conf (hunk 1 above)
Comment (MocanaAtBroad) [reply study-builder/studies/pancan/i18n/en.conf:716]: Maximum length of text exceeded.

Ha excedido el máximo número de caracteres permitidos.

**Review by MocanaAtBroad — CHANGES_REQUESTED** [review]

**Review by ssettipalli — COMMENTED** [review]
validation looks fine but I don't see a patch to apply the validation to already deployed study

**Comment by evgeniipr** [comment]
The PR closed because PR https://github.com/broadinstitute/ddp-study-server/pull/997 for the same changes (patches) is created.

**Pull request closed without merging**